tasks/kubectl-run-acceptance-tests.yml

```
@@ -46,3 +46,14 @@ run:
       --env=RABBITMQ_USER=$(kubectl get secret rabbitmq -o=jsonpath="{.data.rabbitmq-username}" | base64 --decode) \
       --env=RABBITMQ_PASSWORD=$(kubectl get secret rabbitmq -o=jsonpath="{.data.rabbitmq-password}" | base64 --decode) \
       -- /bin/bash -c "sleep 2; behave acceptance_tests/features"
+
+
+      # Run acceptance tests for unaddressed batch
+      kubectl apply -f acceptance-tests-repo/tasks/qid-batch-runner.yml
```

It looks like this yaml is duplicated from the `census-rm-qid-batch-runner` repo. Is it possible to have that as a `get` resource for this task so we don't have to maintain the pod config in two places?

```suggestion
      kubectl apply -f batch-runner-repo/qid-batch-runner.yml
```

---

@liamtoozer I expect batch-runner-repo will need plugging in one of the pipelines in: https://github.com/ONSdigital/census-rm-deploy